tests/conftest.py

```diff
@@ -15,13 +15,13 @@ def flask_test_client():
     from our app, this is a test fixture.
     :return: A flask test client.
     """
-    with create_app().test_client() as test_client:
+    with create_app(testing=True).test_client() as test_client:
         yield test_client
 
 
 @pytest.fixture(scope="session")
 def app():
-    app = create_app()
+    app = create_app(testing=True)
     return app
```

Comment:
I think while here, this maybe should be yield rather than a return? [flask testing docs](https://flask.palletsprojects.com/en/2.1.x/testing/)